pom.xml

```diff
@@ -785,6 +785,12 @@
             <version>5.1.1</version>
           </dependency>
         </dependencies>
+        <configuration>
+          <instructions>
+            <_removeheaders>Bnd-LastModified</_removeheaders>
+            <_removeheaders>Include-Resource</_removeheaders>
```

Comment:
Note that this is the improper syntax. The `-remoteheaders` instruction is a comma delimited value. You should use:
```
<_removeheaders>Bnd-LastModified,Include-Resource</_removeheaders>
```
instead.

Reply:
But you are better off using `-noextraheaders: true` to remove `Bnd-LastModified` (and other build sensitive headers) from the manifest and leave the default for `-removedheaders` alone.

Reply:
Cheers, I fixed the `<_removeheaders>` tag! Regarding the `-noextraheaders: true`, that's what in my first iteration of this PR (b67c7fd53137f43a8051b72df1fa046076c4a5eb), but it [has been suggested to keep the `Tool` header in place](https://github.com/apache/commons-lang/pull/578#issuecomment-656108076), hence the current version only removes `Bnd-LastModified` explicitly. I don't care either way, both solutions would work fine for my purposes.